ietfdata/datatracker.py

```diff
@@ -976,6 +992,21 @@ def meeting_types(self) -> Iterator[MeetingType]:
             url  = meta['next']
             for obj in objs:
                 yield Pavlova().from_mapping(obj, MeetingType)
+                
+     def meeting_room_details(self,id) ->Optional[id]:
```

Comment:
Function should be called `meeting_room()` to match other API calls.
Type for `id` parameter missing. 
Return type of function doesn't match code.